cel/canonical_eventlog.go

```diff
@@ -177,24 +229,24 @@ func unmarshalRecNum(tlv TLV) (uint64, error) {
 	return binary.BigEndian.Uint64(tlv.Value), nil
 }
 
-func createPCRField(pcrNum uint8) TLV {
-	return TLV{pcrTypeValue, []byte{pcrNum}}
+func createIndexField(indexType uint8, indexNum uint8) TLV {
+	return TLV{indexType, []byte{indexNum}}
 }
 
 // UnmarshalPCR takes in a TLV with its type equals to the PCR type value (1), and
```

Comment:
This comment is not longer accurate. It should reflex both PCR and RTMR and the func name is wrong.

Reply:
done